build/openwisp_base/Dockerfile

```diff
@@ -13,33 +15,32 @@ FROM SYSTEM AS PYTHON
 # 1. Twisted: gcc
 # 2. Postgresql: libpq-dev
 RUN apt update && \
-    apt install --yes --no-install-recommends git gcc libpq-dev && \
+    apt install --yes --no-install-recommends gcc libpq-dev && \
```

Comment:
Removed git because it's not required and `git+` installation seem to take more time.